docs/src/main/sphinx/release/release-363.md

```diff
@@ -0,0 +1,67 @@
+# Release 363 (5 Oct 2021)
+
+## General
+
+* Add `http` event listener implementation which sends JSON serialized events to a remote HTTP endpoint. ({issue}`8821`)
+* Improve performance of queries that group by `bigint` columns. ({issue}`9510`)
+* Reduce query memory usage when spill occurs. ({issue}`9270`, {issue}`9275`)
+* Reduce join memory usage. ({issue}`9327`)
+* Fix potential data loss in query results when clients retry requests to the coordinator. ({issue}`9453`)
+* Fix incorrect result for comparisons between zero-valued decimals. ({issue}`8985`)
+* Fix `SHOW ROLES` failure when there are no roles to display. ({issue}`9319`)
+* Fix `EXPLAIN ANALYZE` to show estimates. ({issue}`9396`)
```

Comment:
This sound weird .. 

What did it show before? Maybe reword to 

Show estimated values for EXPLAIN ANALYZE queries